luaui/Widgets/gui_pregameui.lua

```diff
@@ -294,8 +563,22 @@ function widget:Initialize()
 		end
 	end
 
+	if (draftMode == "skill" or draftMode == "random" or draftMode == "fair") then
+		-- auto_ready_disable = true -- if we want to disable auto ready using this mod, otherwise do not uncomment
+		auto_ready_timer = auto_ready_timer * 2 -- extra time to place, just to be safe
+	end
+
+	local xn, zn, xp, zp = Spring.GetAllyTeamStartBox(myAllyTeamID)
+	if xn and (xn ~= 0 or zn ~= 0 or xp ~= msx or zp ~= msz) then
+		hasStartbox = true
+	end
+
 	widget:ViewResize(vsx, vsy)
 	checkStartPointChosen()
+
+	if (draftMode ~= nil and draftMode ~= "disabled") then
+		reloadedDraftMode = os.clock()+2 -- in case you luaui reload
```

Comment:
`reloadedDraftMode` is implicitly global and gets redeclared multiple times

Reply:
It's timer for if you did `/luaui reload` - the widget would ask for the data from the gadget. Also helps if you crashed and were to reconnect.